department-of-veterans-affairs/caseflow
# TYLERB/APPEALS-8964:  Fix Add Issue's Issue Category dropdown not resetting after changing Benefit type dropdown

Resolves [APPEALS-8964](https://vajira.max.gov/browse/APPEALS-8964)

### Description
Made changes to SearchableDropdown so that setting it to an empty string will properly reset the component. Also added a test for it.

### Acceptance Criteria
- [ ] Changing the Benefit type dropdown properly resets the Issue category dropdown in the intake add issue modal

### Testing Plan

1. On the "User selector" text box type "BVAISHAW", click on "BVAISHAW" from the drop down menu and click on blue "Switch User" button.
2. Go to Intake
3. Fill out form until on Add/Remove Issues
4. Click Add an issue
5. Change Issue Category
6. Change Benefit Type
7. Verify that the Issue category is changed back to the default text
8. Select the issue category drop-down and verify that the options are correct for the selected benefit type
